### ISSUE TITLE
Add configuration and improve performance

### ISSUE THREAD

**Issue by TrademarkTM** [bergerhealer/Maplands#7]
When editing a map (zooming in and out, moving..) I noticed that the plugin was at 20% in my timings, which is pretty high and unreliable for players to use. Can't it be optimized or 'delayed' in order to avoid high CPU usage?

When not editing, it seems pretty stable, staying at <1% in timings, which is great. Even so, it would be nice to have a update delay option the configuration. 1 tick seems overkill to me.

Amazing plugin btw! 9/10
Keep up the good work. Thanks!

**Comment by bergerkiller** [comment]
Yeah this should be doable. The idea of maplands was mostly as a static map wall display, but when players are using it as an overhead map I can imagine it being quite laggy. There is already an iterative form of rendering, which means even the processing done in one tick could be spread out over multiple. I'll see if I can come up with a suitable configuration parameter to change the work done per tick.

**Comment by axemwa** [comment]
I'm also having performance issues with this plugin and I'm using 1.13.2 atm, please fix performance
![image](https://user-images.githubusercontent.com/37608633/51756868-e912e100-2111-11e9-8562-4806c60ebe37.png)
Reference Link - https://www.spigotmc.org/threads/maplands.269054/page-2#post-3292580
Timings report # 2 - https://timings.aikar.co/?id=1a365a7281e14538a0592432847f01c7
